coalib/output/Logging.py

```diff
@@ -32,7 +32,7 @@ def get_num_calls_for_level(cls, level):
         return cls._call_counter[level]
 
 
-def configure_logging():
+def configure_logging(no_color=False):
```

Comment:
why not `color` instead of `no_color`? This looks like doing things the other way around ;D